localstack/services/lambda_/invocation/logs.py

```diff
@@ -50,10 +50,32 @@ def run_log_loop(self, *args, **kwargs) -> None:
             log_item = self.log_queue.get()
             if log_item is QUEUE_SHUTDOWN:
                 return
+            logs = log_item.logs.splitlines()
+            # until we have a better way to have timestamps, log events have the same time for a single invocation
+            log_events = [
+                {"timestamp": int(time.time() * 1000), "message": log_line} for log_line in logs
```

Comment:
We will probably be fine, as it should not get reordered based on the timestamp in our codebase. Definitely something to look out for, and fix as soon as possible.

Reply:
😬 🤞 

Not in our Python code, but it could happen when querying from sqlite with CloudWatch v2 🤷 
For example in `localstack/services/cloudwatch/cloudwatch_database_helper.py`, if we `ScanBy.TimestampDescending` 🙈

Reply:
I could increase it in 1ms intervals to be sure :)

Reply:
But reordering Events with the same timestamp would be a parity violation anyway, so not sure how much sense it makes.

Reply:
yeah, but we never had this parity
the closest we have to parity is event ordering (through some random heuristic)

Reply:
guess it doesn't matter too much if we fix it soonish 🔮